awpy/plot/utils.py

```diff
@@ -115,6 +116,69 @@ def is_position_on_lower_level(
         bool: True if the position on the lower level, False otherwise.
     """
     metadata = MAP_DATA[map_name]
-    if position[2] <= metadata["lower_level_max"]:
+    if position[2] <= metadata["lower_level_max_units"]:
         return True
     return False
+
+
+# Old function names:
+def _renaming_warning(old: str, new: str):
+    return f"""Deprecation warning: Function {old} has been renamed to {new}.
+    Please update your code to avoid future deprecation.
+    """
+
+
+def position_transform_axis(
+    map_name: str, position: float, axis: Literal["x", "y"]
+) -> float:
+    """Calls `game_to_pixel_axis` and sends warning.
+    
+    This is the old name of function `game_to_pixel_axis`. Please update
+    your code to avoid future deprecation.
+    """
+    warnings.warn(
+        _renaming_warning("position_transform_axis()", "game_to_pixel_axis()"),
+        DeprecationWarning)
+    return game_to_pixel_axis(map_name, position, axis)
+
+
+def position_revert_axis(
+    map_name: str, position: float, axis: Literal["x", "y"]
+) -> float:
+    """Calls `pixel_to_game_axis` and sends warning.
+    
+    This is the old name of function `pixel_to_game_axis`. Please update
+    your code to avoid future deprecation.
+    """
+    warnings.warn(
+        _renaming_warning("position_revert_axis()", "pixel_to_game_axis()"),
+        DeprecationWarning)
+    return pixel_to_game_axis(map_name, position, axis)
+
+
+def position_transform(
```

Comment:
I also think these functions would still live in `plot`, rather than live in utils (so the import path would be the same)